scrapy/mail.py

```diff
@@ -27,6 +27,8 @@
 from scrapy.utils.misc import arg_to_iter
 from scrapy.utils.python import to_bytes
 
+from email.header import Header
```

Comment:
Please, move this import to the group of imports from the standard library. See [Imports, PEP 8](https://www.python.org/dev/peps/pep-0008/#imports).